src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -5509,7 +5513,6 @@ static void Main()
     }
 }";
             comp = CreateCompilation(sourceB, references: new[] { refA }, parseOptions: TestOptions.RegularPreview, options: TestOptions.ReleaseExe);
-            comp.VerifyDiagnostics();
```

Comment:
Was there a reason to remove this call?

Reply:
> Was there a reason to remove this call?

It was determined redundant in one of the previous PRs due to ```.VerifyDiagnostics()``` below.

---
In reply to: [461201750](https://github.com/dotnet/roslyn/pull/46308#discussion_r461201750) [](ancestors = 461201750)